build.ps1

```diff
@@ -1,80 +1,25 @@
+$ErrorActionPreference = "Stop";
+
 New-Item -ItemType Directory -Force -Path ./nuget
 
 dotnet tool restore
 
-$cd = Get-Location
-
-# storage
-""
-"###########################################"
-"######### IdentityServer4.Storage #########"
-"###########################################"
-""
```

Comment:
Now that the solution is identified in the build output, these headers should not be needed.

![image](https://user-images.githubusercontent.com/677704/66717992-2f62ba00-eddf-11e9-9a86-9bc7e17ac031.png)